### ISSUE TITLE
Should we adhere to the one-sentence-per-line standard [on .tex]?

### ISSUE THREAD

**Issue by maxbiostat** [philsf/trepid_paper#14]
I find it easier. But your emacs 80 character boundary may be preferred.

**Comment by philsf** [comment]
I presume you mean one paragraph per line. What are the pros and cons?

Con: A small change would mean a diff in the whole paragraph, instead of
one or two lines

## 

Enviado do meu celular, desculpe a brevidade.

Felipe Figueiredo
Em 03/06/2015 06:54, "Luiz Max F. Carvalho" notifications@github.com
escreveu:

**Comment by maxbiostat** [comment]
No, I really mean one sentence per line. Meaning we'd break lines at
periods.

On 3 June 2015 at 12:47, Felipe Figueiredo notifications@github.com wrote:

> I presume you mean one paragraph per line. What are the pros and cons?
> 
> Con: A small change would mean a diff in the whole paragraph, instead of
> one or two lines
> 
> ## 
> 
> Enviado do meu celular, desculpe a brevidade.
> 
> Felipe Figueiredo
> Em 03/06/2015 06:54, "Luiz Max F. Carvalho" notifications@github.com
> escreveu:
> 
> > Assigned #14 https://github.com/philsf/trepid_paper/issues/14 to
> > @philsf
> > https://github.com/philsf.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/philsf/trepid_paper/issues/14#event-320978730.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/philsf/trepid_paper/issues/14#issuecomment-108330425.

## 

Luiz Max Fagundes de Carvalho
PhD student, Institute of Evolutionary Biology,  School of Biological
Sciences,
Ashworth Laboratories, Ash 2, office 123
University of Edinburgh, United Kingdom.
http://br.linkedin.com/pub/luiz-max-carvalho/49/687/283

**Comment by philsf** [comment]
Oh. I never thought of that, but it does seem a good idea.

## 

Enviado do meu celular, desculpe a brevidade.

Felipe Figueiredo
Em 03/06/2015 08:54, "Luiz Max F. Carvalho" notifications@github.com
escreveu:

**Comment by philsf** [comment]
Tô read:

http://endlessparentheses.com/longlines-mode-in-latex.html

**Comment by philsf** [comment]
http://fermi.mycloudnas.com/2011/05/fill-hack/
